### PR TITLE
Add UBI9 minimal image support

### PR DESCRIPTION
**Pull request by gdams**
fixes: https://github.com/adoptium/containers/issues/308

### CODE REVIEW

**Comment by tellison** [comment]
Looking.
This won't fully "fix" #308 as there is a test and doc update required too.

**Comment by tellison** [comment]
Requires using `microdnf` ?

**Comment by gdams** [comment]
> This won't fully "fix" https://github.com/adoptium/containers/issues/308 as there is a test and doc update required too.

Perhaps I missed something? where are you looking?

**Comment by tellison** [comment]
> > This won't fully "fix" #308 as there is a test and doc update required too.
> 
> Perhaps I missed something? where are you looking?

Yeah, not part of this PR - just expecting potential changes to container testing in AQAvit set, and update to [web docs](https://adoptium.net/supported-platforms/) etc. We can do those additional tasks under the #308 issue.

File: dockerfile_functions.sh [hunk 1]
```diff
@@ -235,16 +235,16 @@ EOI
 # Select the ubi OS packages.
 print_ubi_pkg() {
 	cat >> "$1" <<'EOI'
-RUN dnf install -y tzdata openssl wget ca-certificates fontconfig glibc-langpack-en gzip tar \
-    && dnf update -y; dnf clean all
+RUN dnf install -y binutils tzdata openssl wget ca-certificates fontconfig glibc-langpack-en gzip tar \
+    && dnf clean all
```
[the remark this answers is not in the document]
Comment (tellison) [reply dockerfile_functions.sh:239]: Ignore me, that's not the "minimal" path

**Review by tellison — APPROVED** [review]

File: dockerfile_functions.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (gdams) [reply dockerfile_functions.sh:239]: that's correct (that's for non-minimal images)

**Review by karianna — APPROVED** [review]
LGTM

**Pull request merged**